core/app/models/concerns/spree/user_address_book.rb

```diff
@@ -76,7 +75,11 @@ def persist_order_address(order)
       # @param default set whether or not this address will show up from
       # #default_address or not
       def save_in_address_book(address_attributes, default = false)
-        return nil unless address_attributes.present?
+        if address_attributes.present?
+          address_attributes = address_attributes.with_indifferent_access
+        else
+          return nil
+        end
```

Comment:
Just for context (please don't consider this question a blocker, as it was already in) why do we accept `{}` for address_attributes?